packages/opentelemetry-exporter/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "opentelemetry-exporter",
+  "version": "1.132.2",
+  "description": "OpenTelemetry Instana Exporter",
+  "keywords": [
+    "opentelemetry",
+    "exporter",
+    "nodejs",
+    "tracing"
+  ],
+  "author": "Willian Carvalho <willian.carvalho@instana.com>",
+  "homepage": "https://github.com/instana/nodejs/blob/main/packages/opentelemetry-exporter/README.md",
+  "license": "MIT",
+  "main": "lib/opentelemetry-exporter.js",
```

Comment:
Looks like the output folder is configured to be `dist` so I think this needs to be `lib/opentelemetry-exporter.js`.

Reply:
Good catch, thanks.
I still did not decide if I will keep TS in there. 
It doesn't make a lot of difference, and it is just "out of the standard" of our code.
So the `main` property will change anyways :)